packages/flutter/test_fixes/cupertino/cupertino.dart

```diff
@@ -286,4 +286,9 @@ void main() {
     inactiveColor: Colors.red,
     fillColor: WidgetStatePropertyAll(CupertinoColors.white),
   );
+
+  // https://github.com/flutter/flutter/pull/161295
+  final cupertinoButton = CupertinoButton(minSize: 60.0);
+  double? minWidth = cupertinoButton.minWidth;
+  double? minHeight = cupertinoButton.minHeight;
```

Comment:
Since the automatic fix is for the constructor, we could probably get rid of the other things here.

```suggestion
  CupertinoButton(minSize: 60.0);
```